.bazelci/presubmit.yml

```diff
@@ -4,22 +4,48 @@ tasks:
   ubuntu1604:
     name: ":linux:"
     shell_commands:
-    - yes | ./configure
+    - wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+    - bash ./Miniconda3-latest-Linux-x86_64.sh -b -p $HOME/miniconda
+    - source $HOME/miniconda/bin/activate
+    - wget https://raw.githubusercontent.com/plaidml/plaidml/master/environment.yml
+    - $HOME/miniconda/bin/conda env update -f ./environment.yml
+    - conda activate $HOME/miniconda
     build_targets:
     - "//plaidml/keras:wheel"
+    build_options:
+    - "--config=linux_x86_64"
     test_targets:
     - "..."
+    test_options:
+    - "--cpu=linux_x86_64"
   macos:
     name: ":apple:"
     shell_commands:
-    - yes | ./configure
+    - curl -O https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh 
+    - sh ./Miniconda3-latest-MacOSX-x86_64.sh -b -p $HOME/miniconda
+    - curl -O https://raw.githubusercontent.com/plaidml/plaidml/master/environment.yml
+    - $HOME/miniconda/bin/conda env update -f ./environment.yml
+    - conda activate $HOME/miniconda
     build_targets:
     - "//plaidml/keras:wheel"
+    build_options:
+    - "--config=windows_x86_64"
     test_targets:
     - "..."
+    test_options:
+    - "--cpu=windows_x86_64"
   windows:
     name: ":window:"
+    batch_commands:
+    - curl https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe -OutFile .\miniconda.exe
```

Comment:
We might want to use a stable version number instead of always just using latest. I've run into bugs with certain version of conda.

Reply:
Miniconda3 4.7.10 is the most recent pinned version that the [conda repos](https://repo.anaconda.com/miniconda/) have. It's also the same thing that latest currently points to. There's also a couple of versions from earlier this year, 4.6.14 and 4.5.12.

Do you think it's a good idea to try running on the Bazel CI machines with latest once as a proof of concept, then pin that version?

Reply:
SGTM